app/nagbot.py

```diff
@@ -116,11 +116,15 @@ def execute_internal(channel, dryrun):
             if len(resources_to_terminate) > 0:
                 message = f'I terminated the following {ec2_type}s: '
                 for r in resources_to_terminate:
-                    contact = sqslack.lookup_user_by_email(r.contact)
-                    message = message + r.make_resource_summary() + \
-                        f', "Terminate after"={r.terminate_after}, "Monthly Price"=' \
-                        f'{util.money_to_string(r.monthly_price)}, Contact={contact}\n'
-                    r.terminate_resource(dryrun=dryrun)
+                    response = r.terminate_resource(dryrun=dryrun)
+                    if response is not True:
+                        message = message + f"Error when attempting to terminate {r.make_resource_summary()}:" \
+                                            f" {response}\n"
+                    else:
+                        contact = sqslack.lookup_user_by_email(r.contact)
+                        message = message + r.make_resource_summary() + \
+                            f', "Terminate after"={r.terminate_after}, "Monthly Price"=' \
+                            f'{util.money_to_string(r.monthly_price)}, Contact={contact}\n'
```

Comment:
You might consider keeping this in the case of failure, it could be handy to have the contact there to help poke 'em